libs/documentation/src/lib/storybook/formly/formly-filter/formly-filter-nested/formly-filter-nested.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+import { FormlyFormOptions, FormlyFieldConfig } from '@ngx-formly/core';
+import { filterFields } from './fields';
```

Comment:
Can you add a tab in the preview to allow the contents of this file to be seen? Take a look here for how to add additional files to preview: https://github.com/GSA/sam-design-system/blob/master/libs/documentation/src/lib/storybook/tree-table/tree-table.stories.ts#L112